src/data/fintype/fin.lean

```diff
@@ -23,6 +23,13 @@ namespace fin
 
 variables {α β : Type*} {n : ℕ}
 
+lemma map_subtype_embedding_univ :
+  (finset.univ : finset (fin n)).map fin.coe_embedding = Iio n :=
```

Comment:
Can you leave a TODO to rename this after the port/perform the rename now already?